wlauto/common/android/device.py

```diff
@@ -670,6 +670,18 @@ def get_device_model(self):
         except KeyError:
             return None
 
+    def is_network_connected(self, network_host='www.google.com'):
+        self.logger.debug('Checking for internet connectivity.')
+        output = self.execute('ping -q -w 1 -c 1 {}'.format(network_host),
```

Comment:
As per sergei's previous comment this should be:

        output = self.execute('{} ping -q -w 1 -c 1 {}'.format(self.busybox, network_host),